### PR TITLE
Document target-scope setting in JSON API

### PR DESCRIPTION
**Pull request by mziolekda**

### CODE REVIEW

File: docs/2.8.11/docs/json-api/index.rst [hunk 1]
```diff
@@ -231,6 +235,15 @@ to issue the request to the Ledger API.
 The HTTP JSON API Service does not validate the token but may need to decode it to extract information that can be used
 to fill in request fields for party-specific request. How this happens depends partially on the token format you are using.
 
+If the Canton participant to which the JSON API is connected uses target-scopes, the configuration has to reflect that
+through the following setting:
```
Comment (carrielaben-da) [inline docs/2.8.11/docs/json-api/index.rst:239]:
```suggestion
A Canton participant connected to the JSON API using target-scopes must be configured as follows:
```

**Review by carrielaben-da — APPROVED** [review]
Slight rewording.

File: docs/2.10.0/docs/json-api/index.rst [hunk 2]
```diff
@@ -231,6 +235,15 @@ to issue the request to the Ledger API.
 The HTTP JSON API Service does not validate the token but may need to decode it to extract information that can be used
 to fill in request fields for party-specific request. How this happens depends partially on the token format you are using.
 
+If the Canton participant to which the JSON API is connected uses target-scopes, the configuration has to reflect that
+through the following setting:
```
[the remark this answers is not in the document]
Comment (mziolekda) [reply docs/2.10.0/docs/json-api/index.rst:239]: I can see how confusing my original words have been. What I meant is that these new settings apply to the JSON API component which in 2.x release line is completely separate process. So if participant has target-scopes in its configuration, also the JSON-API needs them in its configuration.
Can you help me find the right words @carrielaben-da ?

[the remark this answers is not in the document]
Comment (mziolekda) [reply docs/2.10.0/docs/json-api/index.rst:239]: perfect!

Comment (mziolekda) [inline docs/2.10.0/docs/json-api/index.rst:239]: To connect to a Canton participant that uses target-scopes, the JSON API must also include target-scopes in its configuration as follows:

Comment (mziolekda) [inline docs/2.10.0/docs/json-api/index.rst:239]:
```suggestion
To connect to a Canton participant that uses target-scopes, the JSON API must also include target-scopes in its
configuration as follows:
```

**Pull request merged**